### PR TITLE
Task/message list/detailed fragments

### PR DESCRIPTION
**Pull request by teamagam1**
detailed fragments functionality:
text, geo and images 
with goto listener and full screen images.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/65" target="_blank">ReviewNinja

File: app/src/main/java/com/teamagam/gimelgimel/app/view/ImageFullscreenActivity.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        super.onCreate(savedInstanceState);
+
+        ButterKnife.bind(this);
+
+        mImageUri = getIntent().getData();
+
+        // Note that some of these constants are new as of API 16 (Jelly Bean)
+        // and API 19 (KitKat). It is safe to use them, as they are inlined
+        // at compile-time and do nothing on earlier devices.
+        mDraweeView.setSystemUiVisibility(View.SYSTEM_UI_FLAG_LOW_PROFILE
```
Comment (teamagam3) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/ImageFullscreenActivity.java:39]: Could be extracted to method.
I like to crate "init()" method, and categorize every type of initialization. !resolve

File: app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/messags_panel_fragments/MessagesDetailBaseGeoFragment.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    protected GeoMessageInterface mGeoMessageListener;
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+        if (context instanceof GeoMessageInterface) {
+            mGeoMessageListener = (GeoMessageInterface) context;
+        } else {
+            throw new RuntimeException(context.toString()
+                    + " must implement OnFragmentInteractionListener");
```
Comment (teamagam3) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/messags_panel_fragments/MessagesDetailBaseGeoFragment.java:23]: Did you mean GeoMessageInterface? !resolve

**Comment by teamagam3** [comment]
Check you tests.

File: app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/messags_panel_fragments/MessagesDetailBaseGeoFragment.java (hunk 2 above)
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/messags_panel_fragments/MessagesDetailBaseGeoFragment.java:23]: !resolved

File: app/src/main/java/com/teamagam/gimelgimel/app/view/ImageFullscreenActivity.java (hunk 1 above)
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/ImageFullscreenActivity.java:39]: !fixed

**Pull request merged**